compiler/src/main/kotlin/io/spine/protodata/backend/Documentation.kt

```diff
@@ -24,32 +24,37 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata
+package io.spine.protodata.backend
 
-import com.google.protobuf.DescriptorProtos.DescriptorProto
-import com.google.protobuf.DescriptorProtos.EnumDescriptorProto
-import com.google.protobuf.DescriptorProtos.FileDescriptorProto
-import com.google.protobuf.DescriptorProtos.ServiceDescriptorProto
-import com.google.protobuf.DescriptorProtos.SourceCodeInfo.Location
+import com.google.protobuf.DescriptorProtos
+import com.google.protobuf.DescriptorProtos.SourceCodeInfo
 import com.google.protobuf.Descriptors.Descriptor
 import com.google.protobuf.Descriptors.EnumDescriptor
 import com.google.protobuf.Descriptors.EnumValueDescriptor
 import com.google.protobuf.Descriptors.FieldDescriptor
+import com.google.protobuf.Descriptors.FileDescriptor
 import com.google.protobuf.Descriptors.MethodDescriptor
 import com.google.protobuf.Descriptors.OneofDescriptor
 import com.google.protobuf.Descriptors.ServiceDescriptor
+import io.spine.protodata.Doc
 import io.spine.string.trimWhitespace
 import io.spine.util.interlaced
 
 /**
  * Documentation contained in a Protobuf file.
```

Comment:
Given its description, I don't really understand why such a generic thing is now in `.backend`.